tests/unit_tests/physics/test_isotope_kinetic_fractionation_factors.py

```diff
@@ -105,12 +134,68 @@ def test_fig9_values(temperature_C, saturation, alpha):
         alpha_s = formulae.isotope_equilibrium_fractionation_factors.alpha_i_18O(T)
         alpha_k = formulae.isotope_kinetic_fractionation_factors.alpha_kinetic(
             alpha_equilibrium=alpha_s,
-            saturation_over_ice=saturation,
-            diffusivity_ratio_heavy_to_light=diffusivity_ratio_18O(T),
+            saturation=saturation,
+            D_ratio_heavy_to_light=diffusivity_ratio_18O(T),
         )
 
         # act
         sut = alpha_s * alpha_k
 
         # assert
         np.testing.assert_approx_equal(actual=sut, desired=alpha, significant=3)
+
+    @staticmethod
+    @pytest.mark.parametrize("isotope", ("2H", "18O", "17O"))
+    @pytest.mark.parametrize("temperature_C", (-30, -20, -1))
+    def test_alpha_kinetic_jouzel_merlivat_vs_craig_gordon(
+        isotope, temperature_C, plot=PLOT
+    ):
+        # arrange
+        T = Formulae().trivia.C2K(temperature_C)
+        RH = np.linspace(0.3, 1)
+        formulae = Formulae(
+            isotope_equilibrium_fractionation_factors="VanHook1968",
+            isotope_diffusivity_ratios="HellmannAndHarvey2020",
+            isotope_kinetic_fractionation_factors="JouzelAndMerlivat1984",
+        )
+        Si = formulae.saturation_vapour_pressure.pvs_ice(T)
+        alpha_eq = getattr(
+            formulae.isotope_equilibrium_fractionation_factors, f"alpha_l_{isotope}"
+        )(T)
+        D_heavy_to_light = getattr(
+            formulae.isotope_diffusivity_ratios, f"ratio_{isotope}_heavy_to_light"
+        )(T)
+        alpha_kin_jm = formulae.isotope_kinetic_fractionation_factors.alpha_kinetic(
+            alpha_equilibrium=alpha_eq,
+            saturation=Si,
+            D_ratio_heavy_to_light=D_heavy_to_light,
+        )
+        formulae = Formulae(
+            isotope_kinetic_fractionation_factors="CraigGordon",
+        )
+        alpha_kin_cg = formulae.isotope_kinetic_fractionation_factors.alpha_kinetic(
+            relative_humidity=RH,
+            turbulence_parameter_n=1,
+            delta_diff=alpha_eq - 1,
+            theta=1,
+        )
+
+        # act
+        n = (alpha_kin_jm + 1) / (alpha_kin_cg + 1)
+
+        # plot
+        pyplot.plot(1 - RH, n)
+        pyplot.gca().set(
+            xlabel="1-RH",
+            ylabel="turbulence parameter n",
+        )
+        pyplot.grid()
+
+        if plot:
+            show_plot()
```

Comment:
using show_plot() outside of a notebook is misleading (and it creates temporary files)